.github/ISSUE_TEMPLATE/report-compatibility.md

```diff
@@ -0,0 +1,35 @@
+---
+name: "📝 Make a Compatibility Report"
+about: Report Portmaster in/compatibility with Linux Distros, VPN Clients or general Software
+labels: compatibility
+---
+
+**Pre-Submit Checklist**:
+
+- Check applicable sources for existing issues
+  - [Linux Compatibility](https://docs.safing.io/portmaster/install/linux#compatibility)
+  - [VPN Compatibility](https://docs.safing.io/portmaster/install/status/vpn-compatibility)
+  - [Github Issues (all our repos)](https://github.com/issues?q=user%3Asafing)
+
+**Which Linux Distro, VPN Client or General Software is this about?**:
```

Comment:
we must make this easier / clearer what kind of report they are creating. Maybe add all sections and make it easy to remove the ones they are not using. Or have them all in comments and then they just uncomment the one they need?

Reply:
Going with `What in/compatibility are you reporting?`.